File: Programmers/Lv.0/배열의 길이에 따라 다른 연산하기/181854.java
class Solution {
    public int[] solution(int[] arr, int n) {
        int l = arr.length;
        int[] answer = new int[l];
        
        for (int i=0; i<l; i++) {
            int num = arr[i];
            if ((l % 2 == 0 && i % 2 == 1) || (l % 2 == 1 && i % 2 == 0)) num += n;
            answer[i] = num;
        }
        
        return answer;
    }
}

File: Programmers/Lv.0/배열의 길이에 따라 다른 연산하기/181854.js
function solution(arr, n) {
    const l = arr.length;
    const answer = arr.map((v, i) => v + ( (l % 2 == 0 && i % 2 == 1) || (l % 2 == 1 && i % 2 == 0) ? n : 0 ));
    return answer;
}
